ferret/explainers/shap.py

```diff
@@ -40,27 +41,27 @@ def compute_feature_importance(
         target_token_pos_idx = self.helper._check_target_token(text, target_token)
         text = self.helper._check_sample(text)
 
+        # Removing 'target_option' if passed as it's not relevant here
         kwargs.pop('target_option', None)
```

Comment:
I suggest we print a log to the user if we forcefully remove one of their args.

Reply:
Sure, that is a good idea